src/Softeq.NetKit.Chat.Domain.Services/DomainServices/ChannelService.cs

```diff
@@ -328,14 +326,16 @@ public async Task<IReadOnlyCollection<ChannelSummaryResponse>> GetAllowedChannel
                 }
                 else
                 {
-                    if (channelMember.LastReadMessageId.HasValue)
+                    var existingUserChannelMember = await UnitOfWork.ChannelMemberRepository.GetChannelMemberWithMemberDetailsAsync(currentUser.Id, channel.Id);
+
+                    if (existingUserChannelMember.LastReadMessageId.HasValue)
```

Comment:
Possible null reference exception here, let's check on null here

Reply:
not possible, it's already null validation